leapp/cli/upgrade/__init__.py

```diff
@@ -14,6 +17,40 @@
 from leapp.utils.output import report_errors, beautify_actor_exception
 
 
+def archive_logfiles():
+    """ Archive log files from a previous run of Leapp """
+    archive_dir = '/var/log/leapp/archive/'
+    debug_dir = '/var/log/leapp/dnf-debugdata/'
+    leapp_db = '/var/lib/leapp/leapp.db'
+    log_files = [
+        '/var/log/leapp/leapp-upgrade.log',
+        '/var/log/leapp/leapp-report.txt',
+        '/var/log/leapp/dnf-plugin-data.txt'
+    ]
+    files_to_archive = [file for file in log_files if os.path.isfile(file)]
+
+    if os.path.isdir(debug_dir) and os.listdir(debug_dir):
```

Comment:
```suggestion
    if os.path.isdir(debug_dir):
```
This is enough. Because if it is empty, it's not added to the _files_to_archive_ and then it's not deleted.